docs/index.rst

```diff
@@ -18,43 +18,75 @@
 TVM Documentation
 =================
 
+TVM is an open deep learning compiler stack for CPUs, GPUs, and specialized accelerators. It aims to close the gap between the productivity-focused deep learning frameworks, and the performance- or efficiency-oriented hardware backends.
```

Comment:
Can we expand this section a bit? (or are you open for commits doing so?). Eg: what kind of users benefit from TVM, what can I do with it, what benefits does it have over the competition (at a high level).

Reply:
We can discuss whether the content belongs to the landing page (in the main website) or here. One thing that I think we could further improve on are possible pointers(e.g. what are the things that i need to read wearing hat of different personas)